index.md

```diff
@@ -8,29 +8,9 @@ layout: home
 {:.logo}
 ![Optech Logo](/img/logos/optech-notext.png)
 
-The Bitcoin Operations Technology Group (Optech) works to bring the best
-open source technologies and techniques to Bitcoin-using businesses in
-order to lower costs and improve customer experiences.
-
-An initial focus for the group is working with its member organizations to
-reduce transaction sizes and minimize the effect of subsequent transaction fee
-increases.  We provide [workshops][], [documentation][scaling book],
-[weekly newsletters][], [original research][dashboard], [case studies
-and announcements][blog], and help facilitate improved relations between
-businesses and the open source community.
-
-[scaling book]: https://github.com/bitcoinops/scaling-book
-[workshops]: /workshops
-[weekly newsletters]: /en/newsletters/
-[dashboard]: https://dashboard.bitcoinops.org/
-[blog]: /en/blog/
-
-Optech does not exist to make a profit, and all materials and documentation
-produced are released under the MIT license. We are supported by our generous
-founding sponsors and contributions from member companies.
-
-If you're an engineer or manager at a Bitcoin company or an open source contributor and you'd like to be a part of this, please
-contact us at [info@bitcoinops.org](mailto:info@bitcoinops.org).
+Bitcoin Optech works to bring the best open source technologies and techniques
+to Bitcoin-using businesses in order to lower costs and improve customer
+experiences. [Learn more](./about).
```

Comment:
Nitpick: I have a slight preference for "/about" rather than "./about".  Always using absolute paths for internal links is nice because it's context-free when grepping for stuff or doing global substitutions.

Reply:
Fixed!